src/plugins/arduino/ArduinoSketch/ArduinoSketch.ino

```diff
@@ -154,47 +81,87 @@ send_packet(int status_, int value_to_send)
 	Serial.print("\r\n");
 }
 
+inline int
+convert_to_check_sum(int i)
+{
+	//using asci table to convert in to the number it would be casted to
+	byte sum = 0;
+	if (i == 0) {
+		return 48; // if 0 then is it just the aci offset
+	}
+
+	if (i < 0) {
+		sum += 45;  //add the offset for the minus sign
```

Comment:
Why 45 instead of 48? And is there a specific reason for 48 in general?

Reply:
as staed the minus sign is the 45 in asci